.bazelci/format.sh

```diff
@@ -77,13 +77,13 @@ run_proto_formatter () {
     # This is intended to be done by the CI.
     if [[ "$@" == "--check" ]]
     then
-        find . -name '*.proto' -exec $BAZEL run $CLANG_FORMAT -- -i --dry-run --Werror {} +
+        find ~+ -name '*.proto' -exec $BAZEL run $CLANG_FORMAT -- -i -style='{AllowShortFunctionsOnASingleLine: Empty}' --dry-run --Werror {} +
```

Comment:
why are you changing `.` to `~+` here? I don't want to find all files in someone's home directory...

Reply:
Fixed. 
After update it stopped working with relative path so try to update the find command to return the absolute path.

Reply:
After what update? It's nonsensical that `$PWD` would work and `.` would not...

Reply:
With $PWD find returns absolute path while with `.` it returns relative path. With relative path formatter fails with `No such directory` error.

Build result wtih `.`
https://buildkite.com/bazel/buildfarm-farmer/builds/7583#018ca3c0-ab27-4e02-b8db-cb63802da98a/120-149

Build result wtih $PWD
https://buildkite.com/bazel/buildfarm-farmer/builds/7619#018ccdbc-808f-44f8-b312-d01d2dd43d5f/120-149

Reply:
OK, the problem here is that the `bazel run` of clang-format is a bare executable with no inputs. I'm confused as to how it ever worked, since it would always have the working directory of its runfiles. Short of creating a rule that formats with all proto srcs declared as inputs in filegroups, I'm not sure if we'd be able to do better than this.  I'll allow it for now.